Scripts/DCS-BIOS/lib/modules/common_modules/NS430.lua

```diff
@@ -22,15 +21,15 @@ local ns430_aircraft = {
 	"Christen Eagle II",
 	"F-14A-135-GR",
 	"F-14B",
-	"F-16C_50",
+	"F-15C",
+	"F-16A",
```

Comment:
F-16A?

Reply:
Flaming Cliff

Reply:
```
--FC3
add("A-10A", false)
add("F-15C", false)
add("F-16A", false)
add("J-11A", false)
add("MiG-29A", false)
add("MiG-29G", false)
add("MiG-29S", false)
add("Su-25", false)
add("Su-25T", false)
add("Su-27", false)
add("Su-33", false)
```
I took it from this list in AircraftList.lua. So it should be listed under ```mods``` then.

Reply:
Good catch! Yup, should probably be under mods.

Reply:
No idea.

Reply:
Apparently they got it to work in some mod.
https://forum.dcs.world/topic/209209-mb-339apan-by-frecce-tricolori-virtuali-only-compatible-with-dcs-world-256/page/18/#comment-4107636